src/iter/plumbing/mod.rs

```diff
@@ -9,27 +12,84 @@ use super::IndexedParallelIterator;
 use std::cmp;
 use std::usize;
 
+/// The `ProducerCallback` trait is a kind of generic closure,
+/// [analogous to `FnOnce`][FnOnce]. See [the corresponding section in
+/// the plumbing README][r] for more details.
+///
+/// [r]: README.md#producer-callback
```

Comment:
Will this relative link still work on docs.rs and the like?

Reply:
I *think* so...? That's why I made it relative. Could be wrong.

Reply:
Oh, the README.md in particular .. hmm, maybe not. I could link back to the repo.